active/RadialGradientBrush/RadialGradientBrush.md

```diff
@@ -0,0 +1,167 @@
+```
+## Applying banding 
+
+Feathering between colors can be disabled by setting the IsFeatheringEnabled property to false.   
+
+![A picture showing a circle that's fill sharply alternates between black and white.](images/MonochromeRadialGradientBrush.png)
+
+XAML
+```XAML
+<StackPanel>
+  <!-- This rectangle is painted with a radial gradient. -->
+  <Rectangle Width="200" Height="100">
+    <Rectangle.Fill>
+      <RadialGradientBrush EllipseCenter="0.5,0.5" EllipseRadius="0.5,0.5" GradientOrigin="0.5,0.5" IsFeatheringEnabledProperty="False">
+        <GradientStop Color="Black" Offset="0.0" />
+        <GradientStop Color="White" Offset="0.125" />
+        <GradientStop Color="Black" Offset="0.25" />
+        <GradientStop Color="White" Offset="0.375" />
+        <GradientStop Color="Black" Offset="0.5" />
+        <GradientStop Color="White" Offset="1.625" />
+        <GradientStop Color="Black" Offset="0.75" />
+        <GradientStop Color="White" Offset="0.875" />
+        <GradientStop Color="Black" Offset="1.0" />
+      </RadialGradientBrush>
+    </Rectangle.Fill>
+  </Rectangle>
+</StackPanel>
+```
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+### Related articles 
+
+* [AcrylicBrush](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.acrylicbrush)
+* [SolidColorBrush](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Media.SolidColorBrush)
+* [LinearGradientBrush](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Media.LinearGradientBrush)
+* [ImageBrush](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Media.ImageBrush)
+* [WebViewBrush](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.WebViewBrush)
+* [XamlCompositionBrushBase](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.xamlcompositionbrushbase)
+* [Composition brushes](https://docs.microsoft.com/en-us/windows/uwp/composition/composition-brushes)
+
+<!-- ## Recommendations -->
+
+<!-- # API Notes -->
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+```C++
+runtimeclass RadialGradientBrush : Windows.UI.Xaml.Media.GradientBrush
```

Comment:
What's the advantage of deriving from GradientBrush? The reason I ask is that I think we'll actually have to derive from XamlCompositionBrushBase since I think we'd be talking about building a composition effect to do this. So we should just include all the API from that control on here as well.